js/multiFileUpload.js

```diff
@@ -1,351 +1,390 @@
-                uploadDiv.prepend('<div class="error">' + upload.error + '</div>');
-
-                // Remove thumbnail box
-                thumbnailLayer.remove();
-
-                return;
-              }
-
-              thumbnailLayer
-                // Render img tag
-                .html('<img src="' + Qubit.multiFileUpload.uploadTmpDir + '/' + upload.thumb + '"/>')
-
-                // Give thumbnail div a minimum height to prevent text from wrapping to next line
-                .attr('style', function(i) {
-                  return $(this).attr('style') + '; min-length; 100px;'; });
-
-              // Get the file index from the id passed by YUI
-              var fileId = parseHtmlId(event.id);
-
-              // Render final upload
-              $('<div class="form-item">' +
-                  '<label>' + Qubit.multiFileUpload.i18nInfoObjectTitle + '</label>' +
-                  '<input type="text" name="files[' + fileId + '][infoObjectTitle]" value="" style="width: 250px"/>' +
-                  '<input type="hidden" class="md5sum" value="' + upload.md5sum + '"/>' +
-                  '<input type="hidden" class="filename" value="' + upload.name + '"/>' +
-                  '<input type="hidden" name="files[' + fileId + '][name]" value="' + upload.name + '"/>' +
-                  '<input type="hidden" name="files[' + fileId + '][md5sum]" value="' + upload.md5sum + '"/>' +
-                  '<input type="hidden" name="files[' + fileId + '][tmpName]" value="' + upload.tmpName + '"/>' +
-                  '<input type="hidden" name="files[' + fileId + '][thumb]" value="' + upload.thumb + '"/>' +
-                '</div>')
-                .prependTo(thumbnailLayer.next());
-
-              renumerateUploads();
-              highlightRepeatedFiles();
-            });
-
-          uploader.addListener('uploadError', function(event)
-            {
-              log(event);
-
-              var thumbnailLayer = $('#thumbnail-' + event.id);
-
-              // Add error message to progress bar
-              $('div.uploadStatus', thumbnailLayer)
-                .html('<a href="#" class="uploadActionRetry">' + Qubit.multiFileUpload.i18nUploadError + '</a>')
-
-              // Remove cancel button
-              thumbnailLayer.find('a.uploadActionCancel').remove();
-            });
-
-          uploader.addListener('rollOver', function ()
-            {
-              $('#selectLink').addClass('hover');
-            });
-
-          uploader.addListener('rollOut', function ()
-            {
-              $('#selectLink').removeClass('hover');
-            });
-
-          // uploader.addListener('cancel', function () { });
-          // uploader.addListener('click', function () { });
-          // uploader.addListener('mouseDown', function () { });
-          // uploader.addListener('mouseUp', function () { });
-        }
+      return fileName;
     }
-  })(jQuery);
+  };
+
+  $(function ()
+  {
+    var $node = $('body');
```

Comment:
Could we look for a more restrictive element, even if it's added only in this page, starting it after the body doesn't feel right.

Reply:
I have moved this to the .multiFileUpload div.